frontend/src/pages/home/modelCatalog/ModelCatalogSection.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { GalleryItem, Grid, GridItem, PageSection, Stack, StackItem } from '@patternfly/react-core';
+import useDimensions from 'react-cool-dimensions';
+import { conditionalArea, SupportedArea } from '~/concepts/areas';
+import EvenlySpacedGallery from '~/components/EvenlySpacedGallery';
+import ModelCatalogSectionHeader from '~/pages/home/modelCatalog/ModelCatalogSectionHeader';
+import { ModelCatalogCard } from '~/pages/modelCatalog/components/ModelCatalogCard';
+import { ModelCatalogContext } from '~/concepts/modelCatalog/context/ModelCatalogContext';
+import ModelCatalogHint from '~/pages/home/modelCatalog/ModelCatalogHint';
+import { useBrowserStorage } from '~/components/browserStorage';
+import ProjectsLoading from '~/pages/home/projects/ProjectsLoading';
+import { CatalogModel, ModelCatalogSource } from '~/concepts/modelCatalog/types';
+import ModelCatalogSectionFooter from '~/pages/home/modelCatalog/ModelCatalogSectionFooter';
+import { MAX_SHOWN_MODELS, MIN_CARD_WIDTH } from '~/concepts/modelCatalog/const';
+
+const ModelCatalogSection: React.FC = conditionalArea(
+  SupportedArea.MODEL_CATALOG,
+  true,
+)(() => {
+  const { modelCatalogSources } = React.useContext(ModelCatalogContext);
+  const { data, loaded } = modelCatalogSources;
+
+  const models = React.useMemo(
```

Comment:
There is no need to use `useMemo` here because the `models` are not used as any dependencies. So there is no need to memoize it.